torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -823,6 +827,17 @@ def restore_args_from_cpu(self, cpu_copies):
             arg, cpu_arg = pair
             arg.copy_(cpu_arg, non_blocking=True)
 
+    def reset_to_zero_args(self, *args, **kwargs):
+        for i, arg in enumerate(args):
+            if self.fn.arg_names[i] in self.reset_to_zero_arg_names:
+                assert isinstance(arg, torch.Tensor)
```

Comment:
Maybe add an error message to assertion to make it more explicit (i.e., `assert isinstance(arg, torch.Tensor), "..."`)? Ditto below.